marlowe-dashboard-client/src/Capability/Websocket.purs

```diff
@@ -11,7 +11,8 @@ import AppM (AppM)
 import Data.Either (Either(..))
 import Halogen (HalogenM, raise)
 import MainFrame.Types (Msg(..))
-import Plutus.PAB.Webserver.Types (CombinedWSStreamToServer(..))
+import Marlowe.PAB (CombinedWSStreamToServer(..))
+--import Plutus.PAB.Webserver.Types (CombinedWSStreamToServer(..))
```

Comment:
There is no need to leave this comment in, is there? Can clearly see this in the comment. You might want to outline what hack you are applying here in the git commit message though.